src/Azs.InfrastructureInsights.Admin/test/Close-AzsAlert.Tests.ps1

```diff
@@ -1,8 +1,20 @@
-$loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
-if (-Not (Test-Path -Path $loadEnvPath)) {
-    $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+$envFile = 'env.json'
+Write-Host "Loading env.json"
+if ($TestMode -eq 'live') {
+    $envFile = 'localEnv.json'
 }
-. ($loadEnvPath)
+
+if (Test-Path -Path (Join-Path $PSScriptRoot $envFile)) {
+    $envFilePath = Join-Path $PSScriptRoot $envFile
+} else {
+    $envFilePath = Join-Path $PSScriptRoot '..\$envFile'
+}
+$env = @{}
+if (Test-Path -Path $envFilePath) {
+    $env = Get-Content (Join-Path $PSScriptRoot $envFile) | ConvertFrom-Json
+    $PSDefaultParameterValues = @{"*:SubscriptionId" = $env.SubscriptionId; "*:Tenant" = $env.Tenant; "*:Location" = $env.Location; "*:ResourceGroupName" = $env.ResourceGroup }
+}
+
```

Comment:
Please remove this section from all tests and add default parameters to utils.ps1 file as suggested here:
From Xiaogang:

If you need to add a default parameter, you should do it in the setEnv function in utils.ps1.

You may refer to https://github.com/Azure/azure-powershell/blob/generation/src/Databricks/test/utils.ps1 as an example.

If we use this method, then we won't need to add the above lines of code for every test.

Reply:
Putting env setup in utils.ps1 didn't work for me.